message-index/messages/GHC-40798/example1/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: operator-whitespace
+summary: An optional warning for detecting usage of infix, suffix or prefix operators that could be parsed differently in the future due to whitespace, this warning is disabled by default.
```

Comment:
```suggestion
summary: A warning for detecting usage of infix, suffix or prefix operators that could be parsed differently in the future due to whitespace.
```

I personally think the case in which the warning arises is less important -- the very likely case is that the user found this case *because* they have the warning turned on (for whatever reason); it not being default should be covered by the flag field.